libpimeval/src/libpimeval.h

```diff
@@ -146,9 +146,11 @@ PimStatus pimScaledAdd(PimObjId src1, PimObjId src2, PimObjId dest, uint64_t sca
 PimStatus pimPopCount(PimObjId src, PimObjId dest);
 PimStatus pimRedSumInt(PimObjId src, int64_t* sum);
 PimStatus pimRedSumUInt(PimObjId src, uint64_t* sum);
+PimStatus pimRedSumFP32(PimObjId src, float* sum);
```

Comment:
Are we planning to have separate reduction sum APIs for different float types?

Reply:
It depends on our short-term solution, e.g. using float, byte representation, or customized types to support FP8/16